### ISSUE TITLE
LowLevelILFunction.expr default 'flags' argument causes assertion failure

### ISSUE THREAD

**Issue by duck-37** [Vector35/binaryninja-api#2649]
**Binary Ninja Version**
Version 2.4.2997-dev (Build ID 45fa7f79)

**Describe the bug**
When using functions that call `LowLevelILFunction.expr` (i.e. set_reg), the `flags` parameter will be set to `architecture.FlagIndex(0)` if it's not explicitly passed. However, `architecture.FlagIndex` returns an int, which cases `expr` to throw an assertion failure with the message "flags type unsupported."

**To Reproduce**
```
current_llil.append(current_llil.set_reg(8, "rax", current_llil.const(8, 0)))
```
(in python console)

**Expected behavior**
No assertion failure.

**Version and Platform (required):**
 - Binary Ninja: Dev 2.4.2487 (if tested in stable, please also test the development channel via the "update channel" option)
 - OS: Windows 10
 - Version: 21H1 (Build 19043)

**Additional context**
A workaround is simply to pass `ILFlag(architecture, 0)` to whatever function you're calling.

**Comment by duck-37** [comment]
Gah, just noticed https://github.com/Vector35/binaryninja-api/pull/2644/. Dunno if I should close this or not.

**Comment by psifertex** [comment]
It's fine, we'll just make sure to close this too. Never hurts to have an issue to go with the PR. Thanks for filing.

**Comment by plafosse** [comment]
Thanks for reporting this issue it will be fixed in >= 2.4.2998
